pipeline/download_gene.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script that download and postprocess ISH dataset from Allen Brain."""
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+from pathlib import Path
+
+import numpy as np
+import PIL
+
+
+logger = logging.getLogger("download-gene")
+
+
+def parse_args():
+    """Parse command line arguments.
+
+    Returns
+    -------
+    args : argparse.Namespace
+        The parsed command line arguments.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--gene-name",
+        type=str,
+        help="""\
+        Path to CCFv3 annotation volume.
```

Comment:
I think the help is wrong

Reply:
Done in [9bc2e8d](https://github.com/BlueBrain/Deep-Atlas/pull/8/commits/9bc2e8d8ca1a6a9c1ef80f94c923e2d142ec3a84).

Reply:
Thank you